srv/components/s3server/config.sls

```diff
@@ -18,8 +18,6 @@
 Stage - Post Install S3Server:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/s3/conf/setup.yaml', 's3:post_install')
-    - require:
```

Comment:
Let's maintain these require statements to ensure installation.
Any specific reason this is removed here?

Reply:
please check the above response